spec/vehicle_spec.rb

```diff
@@ -33,4 +34,25 @@
       expect(@camaro.electric_vehicle?).to eq(false)
     end
   end
+
```

Comment:
Added tests for helper methods set_registration_date, set_plate_type and set_fees